src/Molecules/UserSelect/UserSelect.test.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+
+import UserSelect from './index'
+
+it('should show label when rendered', () => {
```

Comment:
Missing UT:
- Should display the correct color for a given status
  - Chip
  - MenuItem
- Should remove the item of the selected list
- Should add Item in the selected list